seed/challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -3954,6 +3954,175 @@
+      }, {
+        "text": "'<code>gray(false,381)</code> should return <code>425</code>.'",
+        "testString": "assert.equal(gray(false,381),425,'<code>gray(false,381)</code> should return <code>425</code>.');"
+      }, {
+        "text": "'<code>gray(false,725)</code> should return <code>870</code>.'",
+        "testString": "assert.equal(gray(false,725),870,'<code>gray(false,725)</code> should return <code>870</code>.');"
+      }],
+      "id": "5a23c84252665b21eecc7e80",
+      "challengeType": 5,
+      "releasedOn": "June 7, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": ["function gray (enc,number) {", "  // Good luck!", "}"],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "title": "Greatest common divisor",
+      "description": ["Write a function that returns the greatest common divisor of two integers."],
+      "solutions": ["function gcd(a, b) {\n  return b==0 ? Math.abs(a):gcd(b, a % b);\n}\n"],
+      "tests": [{
+        "text": "'<code>gcd</code> should be a function.'",
+        "testString": "assert(typeof gcd=='function','<code>gcd</code> should be a function.');"
+      }, {
+        "text": "'<code>gcd(24,36)</code> should return a number.'",
+        "testString": "assert(typeof gcd(24,36)=='number','<code>gcd(24,36)</code> should return a number.');"
+      }, {
+        "text": "'<code>gcd(24,36)</code> should return <code>12</code>.'",
+        "testString": "assert.equal(gcd(24,36),12,'<code>gcd(24,36)</code> should return <code>12</code>.');"
+      }, {
+        "text": "'<code>gcd(30,48)</code> should return <code>6</code>.'",
+        "testString": "assert.equal(gcd(30,48),6,'<code>gcd(30,48)</code> should return <code>6</code>.');"
+      }, {
+        "text": "'<code>gcd(10,15)</code> should return <code>5</code>.'",
+        "testString": "assert.equal(gcd(10,15),5,'<code>gcd(10,15)</code> should return <code>5</code>.');"
+      }, {
+        "text": "'<code>gcd(100,25)</code> should return <code>25</code>.'",
+        "testString": "assert.equal(gcd(100,25),25,'<code>gcd(100,25)</code> should return <code>25</code>.');"
+      }, {
+        "text": "'<code>gcd(13,250)</code> should return <code>1</code>.'",
+        "testString": "assert.equal(gcd(13,250),1,'<code>gcd(13,250)</code> should return <code>1</code>.');"
+      }, {
+        "text": "'<code>gcd(1300,250)</code> should return <code>50</code>.'",
+        "testString": "assert.equal(gcd(1300,250),50,'<code>gcd(1300,250)</code> should return <code>50</code>.');"
+      }],
+      "id": "5a23c84252665b21eecc7e82",
+      "challengeType": 5,
+      "releasedOn": "June 7, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": ["function gcd (a,b) {", "  // Good luck!", "}"],
```

Comment:
This one needs a space between the parameters, too:

`"contents": ["function gcd (a,b) {", "  // Good luck!", "}"],`